src/test/java/com/wikia/webdriver/common/remote/operations/Categories.java

```diff
@@ -0,0 +1,50 @@
+package com.wikia.webdriver.common.remote.operations;
+
+import com.jayway.jsonpath.*;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.remote.Discussions;
+import com.wikia.webdriver.common.remote.RemoteException;
+import com.wikia.webdriver.common.remote.context.CreateCategoryContext;
+import com.wikia.webdriver.common.remote.operations.http.GetRemoteOperation;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.category.CategoryPill;
+
+import java.util.ArrayList;
+
+public class Categories {
+
+  public static final String GET_CATEGORIES_URL_SUFFIX = "%s/forums";
+  private final GetRemoteOperation remoteOperation;
+
+  Categories(User user) {
+    remoteOperation = new GetRemoteOperation(user);
+  }
+
+  public ArrayList<CategoryPill.Data> execute(final CreateCategoryContext context) {
+    String response = null;
+    try {
+      response = remoteOperation.execute(buildUrl(context));
+    } catch(RemoteException e) {
+      PageObjectLogging.logError("error: ", e);
+    }
+    return getCategories(response, context);
```

Comment:
Please return early avoiding unnecessary variables or in the catch block if possible.